cli/internal/login/link.go

```diff
@@ -237,6 +233,44 @@ func promptSetup(location string) (bool, error) {
 	return shouldSetup, nil
 }
 
+func addTurboToGitignore() error {
+	fs.EnsureDir(".gitignore")
```

Comment:
This is going to check for the existence of `.`. It's very likely we don't need this line at all.

But, taking this opportunity to explain:
We have an abstraction for absolute system paths (`turbopath.AbsolutePath`) that can help with filesystem operations. You can obtain one of these from the `CmdBase` instance on `link`: `RepoRoot`. From there you can `.Join` to get to the file you want. 

We use those to ensure that we are respecting the `--cwd` flag, which allows users to set what they consider to be the root of the monorepo. So typically, you never want to be using things like `os.ReadFile` directly, use the `AbsolutePath` wrapper around it, which forces us to not rely on the specific directory that `turbo` was invoked in.

Reply:
Makes sense! Do we check beforehand if the directory that's set by `--cwd` exists?